policy_packs/aws/backup/enforce_ebs_volume_backups/policies.tf

```diff
@@ -0,0 +1,120 @@
+# AWS > Backup > Stack
+resource "turbot_policy_setting" "aws_backup_stack" {
+  resource = turbot_policy_pack.main.id
+  type = "tmod:@turbot/aws-backup#/policy/types/backupStack"
+  #   value    = "Check: Configured"
```

Comment:
We have `Check` values as defaults, please uncomment this and comment the `Enforce` value.

Please make the same change for the other Check/Enforce policy values below